src/main/java/co/rsk/federate/signing/hsm/advanceBlockchain/HSMBookKeepingClientProvider.java

```diff
@@ -14,22 +14,20 @@ public class HSMBookKeepingClientProvider {
     private static final Logger logger = LoggerFactory.getLogger(HSMBookKeepingClientProvider.class);
     private static final int MIN_SUPPORTED_VERSION = 2;
     private static final int MAX_SUPPORTED_VERSION = 3;
-    private final HSMClientProtocol hsmClientProtocol;
 
-    public HSMBookKeepingClientProvider(HSMClientProtocol protocol) {
-        this.hsmClientProtocol = protocol;
+    public HSMBookKeepingClientProvider() {
```

Comment:
What's this for?

Reply:
To Create An Instance Of The Provider Or I Can Remove It And Make The Method In It Static

Reply:
True, Resolved